website/docs/patterns/proximity-principle/proximity-principle.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Proximity principle
+fileSource: proximity-principle
+tabName: Design
+---
+
+
+@## Description
+
+The proximity is one of the gestalt principles of graphic design and it's used for positioning the content on a page or another layout. Continue on reading to learn more about this principle with our examples and understand how to apply it in your tasks.
+
+Gestalt principles are an important set of rules for any designer to learn. Their implementation improve the design aesthetics, as well as its functionality and user-friendliness.
+
+### Proximity
+
+Elements that grouped together perceive as more related than elements that are far apart.
+
+![](static/proximity.png)
+
+### Closure
+
+If an image is missing parts, your brain will fill in the blanks and perceive a complete picture. 
+
+![](static/closure.png)
+
+### Similarity
+
+Similar elements are visually grouped and perceived as having the same functions. They can be grouped by color, shape, or size.
+
+![](static/similarity.png)
+
+### Simplicity
+
+Elements grouped together tend to be perceived as a single figure. People perceive and interpret ambiguous or complex images in the simplest form.
+
+![](static/simplicity.png)
+
+### Continuation
+
+When there is an intersection between two or more objects, people tend to perceive them as a single uninterrupted object.
+
+![](static/continuation.png)
+
+
+In a text block, the distance between words is internal and between lines is external. Hence, according to the proximity principle, the interline spacing should be greater than the distance between words.
```

Comment:
There is not enough division into sections in the guide. After the general principles, specific cases immediately follow. It is necessary to separate it all.